app/code/Magento/Catalog/view/frontend/web/js/product/list/toolbar.js

```diff
@@ -34,10 +34,14 @@ define([
 
         /** @inheritdoc */
         _create: function () {
+            if (window.isToolbarLoaded) {
+                return;
+            }
             this._bind($(this.options.modeControl), this.options.mode, this.options.modeDefault);
             this._bind($(this.options.directionControl), this.options.direction, this.options.directionDefault);
             this._bind($(this.options.orderControl), this.options.order, this.options.orderDefault);
             this._bind($(this.options.limitControl), this.options.limit, this.options.limitDefault);
+            window.isToolbarLoaded = true;
```

Comment:
Storing this in a global variable is not needed, you can move it to a local variable above widget declaration.

Reply:
I tried this (this.options.isToolbarLoaded) earlier it is not working as expected so, had to use it as a global variable.

Reply:
I mean adding `var isToolbarLoaded` at line 11 instead: https://github.com/magento/magento2/pull/26097/files#diff-1364732a1dafeccded6675a06411dbdbR11

I also think that changing the wording to `isTollbarInitialized` would be more informative.